cli/pcluster/config/update_policy.py

```diff
@@ -201,3 +201,18 @@ def _check_min_count(change, patch):
     )
     + ". If you need this change, please consider creating a new cluster instead of updating the existing one.",
 )
+
+# Update extra_json
+UpdatePolicy.EXTRA_JSON = UpdatePolicy(
+    level=1000,
+    fail_reason="Updating extra_json will only affect on computefleet and not affect on head node",
```

Comment:
What about something like? 
`Updating the extra_json parameter is not supported because changes would only take place on compute nodes. If you still want to proceed, first stop the cluster with ... and then run an update with the --force flag.`

Reply:
Done